src/T.spec.tsx

```diff
@@ -55,25 +69,6 @@ describe("T", () => {
       });
     });
 
-    describe("When isHTML is false", () => {
```

Comment:
Wait, let me check the code again :)

Reply:
Why do we need to do that?

Reply:
No, the test should still be there. We need to make sure that dangerous code is not injected in placeholders.

Reply:
I propose that we change the test to...

```javascript
expect(container.innerHTML).toEqual("Translated message with placeholder: &lt;script&gt;test&lt;/script&gt;, &lt;iframe src-javascript:alert('s')&gt;");
```

...if this is how it now acts

Reply:
This test was originally added because we had an XXS issue on the site. So I think we should keep it to make sure that we don't end up in that situation again.

Reply:
> We need to make sure that dangerous code is not injected in placeholders to prevent XSS
Hmm, but that impossible if we are not using `dangerouslySetInnerHTML`? Otherwise we would need to sanitize all the form inputs aswell?

Reply:
Or are we using translated stuff in URLs? because then we would definitely need this :) I'll readd it